customization-base/src/main/java/com/azure/autorest/customization/ClassCustomization.java

```diff
@@ -87,6 +87,51 @@ public ClassCustomization addImports(String... imports) {
         return this;
     }
 
+    /**
+     * Adds a static block to the class. The {@code staticCodeBlock} should include the static keyword followed by
+     * the static code.
+     * @param staticCodeBlock The static code block including the static keyword.
+     * @return The updated {@link ClassCustomization}.
+     */
+    public ClassCustomization addStaticBlock(String staticCodeBlock) {
+        return addStaticBlock(staticCodeBlock, null);
+    }
+
+    /**
+     * Adds a static block to the class. The {@code staticCodeBlock} should include the static keyword followed by
+     * the static code.
+     * @param staticCodeBlock The static code block including the static keyword.
+     * @param importsToAdd The list of imports to add to the class.
+     * @return The updated {@link ClassCustomization}.
+     */
+    public ClassCustomization addStaticBlock(String staticCodeBlock, List<String> importsToAdd) {
```

Comment:
Do we want to make static block its own customization type? That way we can support mutating an existing static block.

Reply:
Since there could be multiple static blocks in a class, mutating an existing static block would be equivalent to returning the raw editor. Also, AutoRest doesn't generate any static blocks, so, don't think it's worth creating a new type.

Reply:
Good point and we can always change this in the future